### PR TITLE
Converted Resource List to an Awesome List

### PR DESCRIPTION
**Pull request by Langhalsdino**
Converted Resource List to an Awesome List by:
- Adding batch and table of contents
- Changing name to awesome-landing-pages-resources

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,15 @@
...
+
+## Table of contents
+
+  * [Landing page creation/platform](#Landing-page-creation-platform)
+  * [Analytics](#Analytics)
+  * [Landing Page Themes, Templates, and Plugins](#Landing-Page-Themes-Templates-and-Plugins)
+  * [User Experience, Testing, and Optimization Tools](#User-Experience-Testing-and-Optimization-Tools)
+  * [Blogs](#Blogs)
+  * [Broadcasts and Presentations](#Broadcasts-and-Presentations)
+  * [Books](#Books)
```
Comment (eibrahim) [inline README.md:12]: Awesome... thank you 👍

File: README.md [hunk 2]
```diff
@@ -155,4 +166,3 @@
 
 * [Landing Page Optimization](https://www.google.com/url?q=https://books.google.com.ph/books/about/Landing_Page_Optimization.html?id%3DpnL4JRg7z1cC%26source%3Dkp_cover%26redir_esc%3Dy&sa=D&ust=1507602583004000&usg=AFQjCNFB-z6ANZkwPnBDvUyuGbFuF5pp2Q): A fully updated guide to making your landing pages profitable.
```
Comment (eibrahim) [inline README.md]: can you add the direct link instead of a google redirect.

File: README.md [hunk 3]
```diff
@@ -155,4 +166,3 @@
...
 * [The Conversion Code](https://www.google.com/url?q=https://www.amazon.com/Conversion-Code-Capture-Internet-Appointments/dp/1119211883/ref%3Dsr_1_2?s%3Dbooks%26ie%3DUTF8%26qid%3D1506243860%26sr%3D1-2%26keywords%3Dlanding%2Bpage&sa=D&ust=1507602583005000&usg=AFQjCNE2kE1t3IbYhW7Tws0qZ_N5LWJGbw): Capture and close more Internet leads with a new sales script and powerful marketing templates.
```
Comment (eibrahim) [inline README.md]: can you add the direct link instead of a google redirect.

**Review by eibrahim — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (Langhalsdino) [reply README.md]: will be changed in updated pull request.

File: README.md (hunk 2 above)
Comment (Langhalsdino) [reply README.md]: will be changed in updated pull request.

**Comment by Langhalsdino** [comment]
I changed all google redirect links to direct links.

File: README.md (hunk 1 above)
Comment (Langhalsdino) [reply README.md:12]: You are welcome

**Comment by eibrahim** [comment]
wow... i didn't even realize i had google links.  It's because I converted the original document from a google doc to markdown... thanks a lot.  👍

**Comment by Langhalsdino** [comment]
No problem :)
The following regex might be use full for the next time :
`&sa=D&ust=.*:`
`(.*q=`

**Pull request merged**